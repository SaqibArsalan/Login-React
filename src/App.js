import React, { useState } from "react";
import {BrowserRouter as Router, Redirect, Switch} from "react-router-dom";
import "bootstrap/dist/css/bootstrap.min.css";

import Route from "./components/routes/Routes";
import "./App.css";
import LoginPage from "./login/LoginPage";
import PrivateRoute from "./components/routes/PrivateRoute";
import SignUpPage from "./signup/SignUpPage";


const App = () => {

    return (
        <Router>
            <Switch>
                <Route exact path="/login" component={LoginPage} />
                <Route exact path="/signup" component={SignUpPage} />
                <Redirect from="/" to="/login" component={LoginPage} />
                {/*<PrivateRoute path="/" />*/}
            </Switch>
        </Router>
    );
};

export default App;
