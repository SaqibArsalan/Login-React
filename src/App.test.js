import { render, screen } from '@testing-library/react';
import App from './App';
import { shallow } from "enzyme";

describe("App test", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);
  });
  it("should render the root app component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
