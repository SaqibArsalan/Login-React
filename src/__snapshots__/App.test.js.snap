// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test should render the root app component 1`] = `ShallowWrapper {}`;
